application/bettle/bettle-entity/src/main/java/com/bettle/track/entities/user/UserInfo.java

```diff
@@ -0,0 +1,49 @@
+package com.bettle.track.entities.user;
+
+import com.bettle.track.entities.AbstractParentEntity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.hibernate.annotations.FilterDef;
+import org.hibernate.annotations.NotFound;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+
+/**
+ * UserInfo is an entity of  m_user_info table.
+ * <p>
+ * Create Date: 2017/01/27
+ *
+ * @author Ranjeet Parashar
+ * @version 1.0
+ * @since 1.0
+ */
+
+@Entity
+@Table(name = "m_user_info")
+@FilterDef(name = "active_user_info", defaultCondition = "active_flag = 1")

```

Comment:
Add @FilterDefs and keep @FilterDef in @FilterDefs as array.
Add @Where above @FilterDefs annotation.